packages/abi-typegen/src/templates/contract/factory.test.ts

```diff
@@ -24,6 +25,8 @@ describe('templates/factory', () => {
     const project = getTypegenForcProject(AbiTypegenProjectsEnum.MINIMAL);
     const rawContents = project.abiContents;
 
+    vi.spyOn(utilsMod, 'compressBytecode').mockReturnValueOnce('0x-bytecode-here');
```

Comment:
Is this mock making a difference, and should we assert this value?

Reply:
Also, now that the string is not a hex string anymore, should we rename the variable?

From `hexlifiedBinContents` to _______?

Reply:
@arboleya yes, I mocked it because the output of the `compressBytecode` function is not deterministic

And I have already renamed the variable on the template to `compressedBytecode` if that's what you are referring to?